frontend/src/utilities/useWatchBuildStatus.tsx

```diff
@@ -44,7 +44,7 @@ export const useWatchBuildStatus = (): BuildStatus[] => {
     };
   }, []);
 
-  const buildStatuses = useDeepCompareMemoize<BuildStatus[]>(statuses);
+  const buildStatuses = useDeepCompareMemoize<BuildStatus[] | undefined>(statuses);
```

Comment:
Not possible to be `undefined`.
```suggestion
  const buildStatuses = useDeepCompareMemoize(statuses);
```